docs/networking.rst

```diff
@@ -57,7 +57,7 @@ The configuration to use an existing private network requires the following sett
   master_subnet_id = subnet-<public>
   compute_subnet_id = subnet-<private>
 
-Both these configuration require to have a `NAT Gateway
+Note that this second configuration require to have a `NAT Gateway
```

Comment:
Could you please verify if it works by using `use_public_ips = false`?

Reply:
it does not work.

Reply:
require --> requires

Reply:
Thank you.
We could use the `..note::` syntax to show the message as an annotation:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html

Then: require --> requires

Reply:
Done